neuralmonkey/config/builder.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, string):
 
     def create(self):
         class_parts = self.clazz.split(".")
+
```

Comment:
jo `clazz` neni objekt třídy ale string.. No tak to bych ještě přejmenoval `clazz`.. :-)